karma.conf.js

```diff
@@ -0,0 +1,106 @@
+module.exports = function(config) {
+
+  'use strict';
+
+  // https://saucelabs.com/platforms
+  var customLaunchers = {
+    sl_chrome: {
+      base: 'SauceLabs',
+      browserName: 'chrome',
+      version: '51'
+    },
+    sl_firefox: {
+      base: 'SauceLabs',
+      browserName: 'firefox',
+      version: '47'
+    },
+    sl_Safari_8: {
+      base: 'SauceLabs',
+      browserName: 'safari',
+      platform: 'OS X 10.10',
+      version: '8'
+    },
+    SL_Safari_10: {
+      base: 'SauceLabs',
+      browserName: 'safari',
+      platform: 'OS X 10.11',
+      version: '10'
+    },
+    SL_iOS_Safari_8: {
+      base: 'SauceLabs',
+      browserName: 'iphone',
+      version: '8.1'
+    },
+    SL_IE_9: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      platform: 'Windows 2008',
+      version: '9'
+    },
+    sl_ie_11: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      platform: 'Windows 8.1',
+      version: '11'
+    }
+  };
+
+  config.set({
+
+    browserDisconnectTimeout: 10000,
+    browserDisconnectTolerance: 2,
+    browserNoActivityTimeout: 90000,
+    captureTimeout: 120000,
+
+    client: {
+      mocha: {opts: 'test/mocha.opts'}
+    },
+
+    frameworks: [
+      'browserify',
+      'mocha'
+    ],
+
+    files: [
+      './node_modules/ramda/dist/ramda.js',
+      './node_modules/sanctuary-def/index.js',
+      'index.js',
+      'test/**/*.js'
+    ],
+
+    preprocessors: {
+      'test/**/*.js': ['browserify']
+    },
+
+    reporters: [
+      'dots',
+      'saucelabs'
+    ],
+
+    singleRun: true,
+
+    browserify: {
+      configure: function(bundle) {
+        bundle.on('prebundle', function() {
+          bundle.require(
+            './node_modules/ramda/dist/ramda.js',
+            {expose: 'ramda'});
+          bundle.require(
+            './node_modules/sanctuary-def/index.js',
+            {expose: 'sanctuary-def'});
```

Comment:
I'd prefer this formatting:

```javascript
bundle.require('./node_modules/ramda/dist/ramda.js',
               {expose: 'ramda'});
bundle.require('./node_modules/sanctuary-def/index.js',
               {expose: 'sanctuary-def'});
```

Actually, since we're referencing these paths more than once we could define them as "constants" near the top of the file, giving:

```javascript
bundle.require(ramdaMain, {expose: 'ramda'});
bundle.require(sanctuaryDefMain, {expose: 'sanctuary-def'});
```